docs/plugins/ingest-attachment.asciidoc

```diff
@@ -238,3 +238,58 @@ properties on this top level field will contain the value of the
 first attachment only. However, by specifying the
 `target_field` on to a value on `_ingest._value` it will correctly
 associate the properties with the correct attachment.
+
+[[ingest-attachment-raw-metadata]]
+==== Extracting all raw metadata from documents
+
+By default, the `ingest-attachment` plugin only extracts a subset of the most common metadata.
+
+If you want to get back all the raw metadata, you can set `properties` to `raw_metadata`.
+It will populate a subfield `raw_metadata` with all key/value pairs found as metadata in the document.
+
+For example:
+
+[source,js]
+--------------------------------------------------
+PUT _ingest/pipeline/attachment
+{
+  "description" : "Extract all metadata",
+  "processors" : [
+    {
+      "attachment" : {
+        "field" : "data",
+        "properties": [ "raw_metadata" ]
+      }
+    }
+  ]
+}
+PUT my_index/my_type/my_id?pipeline=attachment
+{
+  "data": "e1xydGYxXGFuc2kNCkxvcmVtIGlwc3VtIGRvbG9yIHNpdCBhbWV0DQpccGFyIH0="
+}
+GET my_index/my_type/my_id
+--------------------------------------------------
+// CONSOLE
+
+Returns this:
+
+[source,js]
+--------------------------------------------------
+{
+  "found": true,
+  "_index": "my_index",
+  "_type": "my_type",
+  "_id": "my_id",
+  "_version": 1,
+  "_source": {
+    "data": "e1xydGYxXGFuc2kNCkxvcmVtIGlwc3VtIGRvbG9yIHNpdCBhbWV0DQpccGFyIH0=",
+    "attachment": {
+      "raw_metadata": {
```

Comment:
Yes we can change that but I think it can be easier for users to separate that. So they can more easily ignore the whole content of this inner object at index time with `enabled: false` in the mapping.